src/composables/useTokenGenerator/useTokenGenerator.test.ts

```diff
@@ -18,12 +19,12 @@ describe('useTokenGenerator Composable', () => {
         const runs = 100;
         for (let index = 0; index < runs; index++) {
             const token = generateValidToken();
-            expect(rules.required(token)).toBe(true);
-            expect(rules.min(token)).toBe(true);
-            expect(rules.uppercase(token)).toBe(true);
-            expect(rules.lowercase(token)).toBe(true);
-            expect(rules.special(token)).toBe(true);
-            expect(rules.number(token)).toBe(true);
+            expect(defaultRules.required(token)).toBe(true);
+            expect(defaultRules.min(token)).toBe(true);
+            expect(defaultRules.uppercase(token)).toBe(true);
+            expect(defaultRules.lowercase(token)).toBe(true);
+            expect(defaultRules.special(token)).toBe(true);
+            expect(defaultRules.number(token)).toBe(true);
```

Comment:
why that name change?

Reply:
initially wanted to make the generateValidToken so it could recieve an different ruleset but wasn't sure if we needed that so i left it out, and forgot discard the name change

Reply:
aight fine